src/stickybits.js

```diff
@@ -186,7 +186,7 @@ Stickybits.prototype.computeScrollOffsets = function computeScrollOffsets (item)
   const parent = it.parent
   const isCustom = !this.isWin && p.positionVal === 'fixed'
   const isBottom = p.verticalPosition !== 'bottom'
-  const scrollElOffset = isCustom ? p.scrollEl.getBoundingClientRect().top : 0
+  const scrollElOffset = isCustom ? document.querySelector(p.scrollEl).getBoundingClientRect().top : 0
```

Comment:
^ Nice catch! 😍 

Could this be in another Pull Request? 

Also, could this be on [line line 61](https://github.com/DanielRuf/stickybits/blob/08f63e94ac9e63fe17b58373f25c0f03c31dc57b/src/stickybits.js#L61)?

🙏🙏🙏

Reply:
Sure, will do it later today.

Reply:
@DanielRuf 🙏